x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/configuration/ConfigurationValidationType.java

```diff
@@ -14,8 +14,7 @@ public enum ConfigurationValidationType {
     GREATER_THAN,
     LIST_TYPE,
     INCLUDED_IN,
-    REGEX,
-    UNSET;
```

Comment:
The `UNSET` is used as null placeholder in [connectors framework](https://github.com/elastic/connectors/blob/225592b0823a895fc8cff7a7f4738232720993d5/connectors/source.py#L75), it doesn't have any meaning and should not be considered a valid enum. Since we have list of validations for a given config value, empty list implies no validations.